src/main/resources/org/spongepowered/launch/ivysettings.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<ivysettings>
+    <settings defaultResolver="sponge" />
+    <resolvers>
+        <chain name="sponge">
+			<ibiblio name="m2" m2compatible="true" />
```

Comment:
our repo should be added here and central should NOT be in this list at all... all libraries that are not hosted by mojang itself that are needed are mirrored in our repository